tensorflow/tf_version_script.lds

```diff
@@ -2,6 +2,7 @@ tensorflow {
   global:
     *tensorflow*;
     *perftools*gputools*;
+    TF_*;
```

Comment:
I confirmed that `--exclude_symbols_list` is case sensitive by replacing `TF_` with `*tf_*` and observing that the `TF_New*` symbols were present with `TF_` but absent with `*tf_*`.